src/chalk-overview.md

```diff
@@ -0,0 +1,141 @@
+# An Overview of Chalk
+
+> Chalk is under heavy development, so if any of these links are broken or if
+> any of the information is inconsistent with the code or outdated, please open
+> an issue so we can fix it. If you are able to fix the issue yourself, we would
+> love your contribution!
+
+[Chalk][chalk] recasts Rust's trait system explicitly in terms of logic
+programming by "lowering" Rust code into a kind of logic program we can then
```

Comment:
Perhaps a link to the lowering chapters would be good here?
- https://rust-lang-nursery.github.io/rustc-guide/traits-lowering-to-logic.html
- https://rust-lang-nursery.github.io/rustc-guide/traits-lowering-rules.html